magister-calendar.js

```diff
@@ -392,9 +393,14 @@ function parseAppointments(appointments, currentcourse) {
     }
     /* End of special code block. */
 
+    // Personal appointments don't have a schoolhour assigned. This makes sure it will show a 'P' (for personal) instead of 'null'
+    if(appointment.schoolhour == null){
```

Comment:
When a school hour is not set (null), the appointment is not automatically a personal appointment.

For example, the appointment could be an all-day activity, or a reminder set by your school. I'm not sure if there's a way to detect the difference.